Clock/PyQtPiClock.py

```
@@ -23,13 +23,146 @@
+        return
+    try:
+        # Expect to be run from Clock/ (startup.sh does cd Clock)
+        log_file = os.path.join(os.getcwd(), "PyQtPiClock.1.log")
+        logger = _DailyRotatingLineLogger(log_file, keep=7, tee_to=None)
+        sys.stdout = logger
+        sys.stderr = logger
+        import atexit
+        atexit.register(logger.close)
+    except Exception:
+        # If anything goes wrong, fall back to normal stdout/stderr.
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)